gear/csv_center_splitter/src/docker/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "csv-center-splitter",
+    "label": "CSV Center Splitter",
+    "description": "Splits a CSV by center ID",
+    "version": "0.1.0",
+    "author": "NACC",
+    "maintainer": "NACC <nacchelp@uw.edu>",
+    "cite": "",
+    "license": "MPL-2.0",
+    "url": "https://naccdata.github.io/flywheel-gear-extensions",
+    "source": "https://github.com/naccdata/flywheel-gear-extensions",
+    "environment": {
+        "FLYWHEEL": "/flywheel/v0"
+    },
+    "custom": {
+        "gear-builder": {
+            "category": "utility",
+            "image": "naccdata/csv-center-splitter:0.1.0"
+        },
+        "flywheel": {
+            "suite": "Curation",
+            "show-job": true
+        }
+    },
+    "inputs": {
+        "api-key": {
+            "base": "api-key"
+        },
+        "input_file": {
+            "description": "The input CSV",
+            "base": "file",
+            "type": {
+                "enum": [
+                    "tabular data"
+                ]
+            }
+        }
+    },
+    "config": {
+        "dry_run": {
+            "description": "Whether to do a dry run",
+            "type": "boolean",
+            "default": false
+        },
+        "adcid_key": {
+            "description": "Column name from the input CSV with the ADCIDs",
+            "type": "string"
+        },
+        "target_project": {
+            "description": "Target project name to write the split results to; each ADCID in the split results is expected to have a project",
+            "type": "string",
+            "default": ""
+        },
+        "staging_project_id": {
+            "description": "Staging project ID to stage all results to; overrides target_project",
+            "type": "string",
+            "default": ""
+        },
+        "allow_merged_cells": {
```

Comment:
Ah okay - I added it because some of the NCRAD files have merged cells, which end up looking like this when converted to CSV:

<img width="1378" alt="image" src="https://github.com/user-attachments/assets/a215add9-1437-494e-a4cd-34bb76480552" />

I think there was a conversation saying it was fine to leave them like that but I could be misremembering/we can ask NCRAD to fill in the blanks. Either way I'll remove it.

Reply:
they aren't supposed to be doing that